pkgs/development/tools/database/mermerd/default.nix

```
@@ -0,0 +1,29 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "mermerd";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "KarnerTh";
+    repo = "mermerd";
+    rev = "v${version}";
+    sha256 = "sha256-mQIHv3KjUCtXhzbqBNuooCH7b/LXVUHTZO5n0CJym/A=";
```

SRI hashes are preferred nowadays.

```suggestion
    hash = "sha256-mQIHv3KjUCtXhzbqBNuooCH7b/LXVUHTZO5n0CJym/A=";
```